docs/transforms.md

```diff
@@ -33,7 +33,7 @@ There are 3 types of transforms: attribute, name, and value.
 
 **Attribute:** An attribute transform adds to the attributes object on a design token. This is for including any meta-data about a design token such as it's CTI or other information.
 
-**Name:** A name transform transform the name of a design token. You should really only be apply one name transformer because they will override each other if you use more than one.
+**Name:** A name transform transforms the name of a design token. You should really only be applying one name transformer because they will override each other if you use more than one.
```

Comment:
Can you update this file: https://github.com/amzn/style-dictionary/blob/main/scripts/handlebars/templates/transforms.hbs  the docs/transforms.md file is generated via a script based on that handlebars file

Reply:
Done!